src/main/java/fi/hsl/transitdata/cancellation/util/TimeUtils.java

```diff
@@ -1,16 +1,27 @@
 package fi.hsl.transitdata.cancellation.util;
 
+import org.apache.commons.lang3.time.DateUtils;
+import org.apache.commons.lang3.time.DurationFormatUtils;
+import org.joda.time.DateTimeUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.time.Instant;
 import java.time.LocalDateTime;
 import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 
 public class TimeUtils {
 
     private static final Logger log = LoggerFactory.getLogger(TimeUtils.class);
-
+    
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd HHmm");
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd");
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HHmm");
+    
     // This methods is copied from transitdata-omm-cancellation-source
```

Comment:
How is these methods are UTC? Clearly should be renamed since it takes timezone as parameter

Reply:
If you mean two methods toUtcEpochMs(...), I removed them because they were not used.